File: ios/coffee-oc/coffee/coffee/DataModels/Enums.m
//
//  Enums.m
//  coffee
//
//  Created by achiel on 14-10-6.
//  Copyright (c) 2014年 1clip. All rights reserved.
//

#import "Enums.h"

@implementation Gender

+(NSString*) Male
{
    return @"male";
}

+(NSString*) Female
{
    return @"female";
}

@end

@implementation HangoutState

+(NSString*) Active
{
    return @"active";
}

+(NSString*) Overdue
{
    return @"overdue";
}

@end

@implementation ParticipatorRole

+(NSString*) Organizer
{
    return @"organizer";
}

+(NSString*) Invitee
{
    return @"invitee";
}

@end

@implementation ParticipatorState

+(NSString*) Pending
{
    return @"pending";
}

+(NSString*) Accept
{
    return @"accept";
}

+(NSString*) Reject
{
    return @"reject";
}

@end

File: ios/coffee/coffee/Data Models/Enums.swift
//
//  Enums.swift
//  coffee
//
//  Created by achiel on 14-10-4.
//  Copyright (c) 2014年 1Clip. All rights reserved.
//

// CAUSION: JSONModel does not support enum currently.
//          Use string for enum property in various classes.

// User enumerations
public enum GENDER : String {
    case Male = "male"
    case Female = "female"
}

// Hangout enumerations
public enum HangoutState : String {
    case Active = "active"
    case Overdue = "overdue"
}

public enum ParticipatorRole : String {
    case Organizer = "organizer"
    case Invitee = "invitee"
}

public enum ParticipatorState : String {
    case Pending = "pending"
    case Accept = "accept"
    case Reject = "reject"
}



